pkg/sfu/buffer/buffer_test.go

```diff
@@ -59,11 +60,17 @@ func TestNack(t *testing.T) {
 			HeaderExtensions: nil,
 			Codecs:           []webrtc.RTPCodecParameters{vp8Codec},
 		}, vp8Codec.RTPCodecCapability, Options{})
-		buff.nacker.SetRTT(0)
+		rtt := uint32(20)
+		buff.nacker.SetRTT(rtt)
 		for i := 0; i < 15; i++ {
 			if i == 1 {
 				continue
 			}
+			if i < 14 {
+				time.Sleep(time.Duration(float64(rtt)*math.Pow(backoffFactor, float64(i))+10) * time.Millisecond)
```

Comment:
Does this will cause test cost too long to complete?

Reply:
Yeah, it increases test by about 5 seconds.